examples/CRISP/server/.env.example

```
@@ -15,9 +15,9 @@ CRON_API_KEY=1234567890
 
 # Enclave stack: updated automatically on each `pnpm dev:up` deploy (do not edit by hand unless debugging).
 # Stale E3_PROGRAM_ADDRESS causes requestE3 to revert with empty data `0x`.
-ENCLAVE_ADDRESS=0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0
+ENCLAVE_ADDRESS=0x9A676e781A523b5d0C0e43731313A708CB607508
 FEE_TOKEN_ADDRESS=0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
-E3_PROGRAM_ADDRESS=0x162A433068F51e18b7d13932F27e66a3f99E6890
+E3_PROGRAM_ADDRESS=0xc351628EB244ec633d5f21fBD6621e1a683B1181
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E3_PROGRAM_ADDRESS key should go before the ENCLAVE_ADDRESS key

(UnorderedKey)